src/object-store/src/manager.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use snafu::ensure;
+
+use crate::error::Result;
+use crate::{error, ObjectStore};
+
+/// Manages multiple object stores so that users can configure a storage for each table.
+/// This struct certainly have one default object storage, and can have zero or more object stores.
+pub struct ObjectStoreManager {
+    stores: HashMap<String, ObjectStore>,
+    default_object_store: String,
```

Comment:
LGTM, but I'd like to retrieve the default `ObjectStore` in `new`, and simplify the `default_object_store` to return it directly.

Reply:
I guess that I addressed your review: [28e8134](https://github.com/GreptimeTeam/greptimedb/pull/2621/commits/28e813422d179af09e12bed1207ee3b9d8afea5a)